cupyx/scipy/fft/_fft.py

```diff
@@ -571,3 +572,113 @@ def ihfft(x, n=None, axis=-1, norm=None, overwrite_x=False, *, plan=None):
     if plan is not None:
         raise NotImplementedError('ihfft plan is currently not yet supported')
     return _ihfft(x, n, axis, norm)
+
+
+@_implements(_scipy_fft.hfft2)
+def hfft2(x, s=None, axes=(-2, -1), norm=None, overwrite_x=False, *,
+          plan=None):
+    """Compute the FFT of a two-dimensional signal that has Hermitian symmetry.
+
+    Args:
+        x (cupy.ndarray): Array to be transformed.
+        s (None or tuple of ints): Shape of the real output.
+        axes (tuple of ints): Axes over which to compute the FFT.
+        norm (``"backward"``, ``"ortho"``, or ``"forward"``): Optional keyword
+            to specify the normalization mode. Default is ``None``, which is
+            an alias of ``"backward"``.
+        overwrite_x (bool): If True, the contents of ``x`` can be destroyed.
+        (This argument is currently not supported)
```

Comment:
This sentence needs to be indented so that Sphinx can render it correctly:
https://cupy--4996.org.readthedocs.build/en/4996/reference/generated/cupyx.scipy.fft.hfft2.html#cupyx.scipy.fft.hfft2

Reply:
Oops! I'll fix this